compiler/rustc_typeck/src/check/fn_ctxt/checks.rs

```diff
@@ -1790,3 +1725,47 @@ impl<'a, 'tcx> FnCtxt<'a, 'tcx> {
         }
     }
 }
+
+fn label_fn_like<'tcx>(
```

Comment:
Can you add a comment for this function, please?

Reply:
This was in #96155